templates/user/profile.tmpl

```diff
@@ -31,7 +31,7 @@
 							{{if .Owner.Website}}
 								<li>
 									<i class="octicon octicon-link"></i>
-									<a target="_blank" rel="noopener noreferrer" href="{{.Owner.Website}}">{{.Owner.Website}}</a>
+									<a target="_blank" rel="noopener noreferrer me" href="{{.Owner.Website}}">{{.Owner.Website}}</a>
```

Comment:
Should we add "nofollow" as well as suggested in https://github.com/gogs/gogs/issues/5008?

Reply:
Sorry for the delay. Done :)